db/src/commonMain/sqldelight/com/islamversity/db/tables.sq

```diff
@@ -0,0 +1,101 @@
+import com.islamversity.db.model.EntityUUID;
+import com.islamversity.db.model.Calligraphy;
+
+CREATE TABLE IF NOT EXISTS calligraphy (
```

Comment:
add 'name' property for calligraphy tale

Reply:
name?
why do we have name in calligraphy?